entrypoint.sh

```diff
@@ -63,4 +63,10 @@ helm dependency update .
 
 # Using the repo name as the chart name and not the chart folder name
 PACKAGE=$(helm package --app-version $CHART_APP_VERSION --version $CHART_VERSION . | rev | cut -d'/' -f1 | rev)
-curl -u "$CHARTMUSEUM_USER:$CHARTMUSEUM_PASSWORD" "$CHARTMUSEUM_URL" --upload-file "$PACKAGE"
\ No newline at end of file
+if [ -z "$CHARTMUSEUM_URL" == "https://chartmuseum.devops.bestegg.com/"]; then
```

Comment:
I think you accidentally have a `-z` in there

Reply:
Thanks for finding that, I just updated that now